src/main/java/net/sf/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -126,8 +127,8 @@ private OpenOfficePanel() {
         update = new JButton(IconTheme.JabRefIcon.REFRESH.getSmallIcon());
         update.setToolTipText(Localization.lang("Sync OpenOffice/LibreOffice bibliography"));
         preferences = new OpenOfficePreferences(Globals.prefs);
-        loader = new StyleLoader(preferences, Globals.prefs, Globals.journalAbbreviationLoader,
-                Globals.prefs.getDefaultEncoding());
+        loader = new StyleLoader(preferences, LayoutFormatterPreferences.fromPreferences(Globals.prefs),
```

Comment:
So if the OOPanel is open and I make some changes to the preferences, then `StyleLoader` is not changed, i.e. every action from the OOPanel still uses the old preferences, right?